src/auxiliary/mock_turbo.jl

```diff
@@ -0,0 +1,37 @@
+# Copies some of the LoopVectorization functionality, 
+# but solely using Julia base functionality. It is equivalent to `@simd`
+# at every loop level
+macro turbo(exprs...)
```

Comment:
Does this work with something like https://github.com/trixi-framework/Trixi.jl/blob/5acf73a96f4d4e72cb72169ede148c8d106838fb/src/solvers/dgmulti/types.jl#L361? Asked differently, will this throw a runtime error or something else/?

Reply:
This will work, but will not be threaded. Will essentially turn into `@simd for i`

Reply:
Thanks!